rocket/rocket.py

```diff
@@ -54,49 +53,53 @@ def launch(
             project_location: str = ".",
             dbfs_path: Optional[str] = None,
             watch: bool = True,
-            glob_path: Optional[Union[str, List[str]]] = None
+            glob_path: Optional[Union[str, List[str]]] = None,
+            use_volumes: Optional[bool] = False,
```

Comment:
Is this needed?

Reply:
Do you mean `use_volumes`? we don't want to change the current behaviour but only add an option for using volumes.